inspirehep/modules/workflows/workflows/core_selection.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+from __future__ import absolute_import, division, print_function
+
+from inspirehep.modules.literaturesuggest.tasks import curation_ticket_context
+from inspirehep.modules.records.errors import MissingInspireRecordError
+from inspirehep.modules.workflows.tasks.actions import halt_record
+from inspirehep.modules.workflows.tasks.submission import prepare_keywords, create_ticket
+from inspirehep.modules.workflows.tasks.upload import store_record
+from inspirehep.modules.workflows.utils import get_record_from_hep, do_not_repeat
+
+
+def load_record_from_hep(obj, eng):
+    control_number = obj.data['control_number']
+    record_data = get_record_from_hep("lit", control_number)
+    if not record_data or 'metadata' not in record_data:
+        raise MissingInspireRecordError
+    obj.data = record_data['metadata']
+    return obj
+
+
+def set_core(obj, eng):
+    obj.data['core'] = True
+    return obj
+
+
+class CoreSelection(object):
+    """Workflow for changing coreness of the paper"""
+    name = "CORE_SELECTION"
+    data_type = "hep"
+
+    workflow = (
+        halt_record(
+            action='auto_non_core_record',
+            message='Submission halted Waiting for curator to decide if record is CORE.'
+        ),
+        load_record_from_hep,
+        set_core,
+        prepare_keywords,
```

Comment:
we'll need to do also the affiliation normalization from #4043.

Reply:
It's not merged yet so I can't do anything about this right now.

Reply:
It is now :)

Reply:
a new one to add: https://github.com/inspirehep/inspire-next/blob/b2582d2cc0c3e9d24b3783f4fcc529af3d8b83b7/inspirehep/modules/workflows/workflows/article.py#L75